Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -643,12 +664,445 @@ def resume_scan_command():
+    human_readable = []
+    for chunk_details in chunks_details_list:
+        asset_details = chunk_details.get('asset')
+        plugin_details = chunk_details.get('plugin')
+        port_details = chunk_details.get('port')
+        human_readable_to_append = {}
+        if asset_details:
+            human_readable_to_append['ASSET ID'] = asset_details.get('uuid')
+            human_readable_to_append['ASSET NAME'] = asset_details.get('hostname')
+            human_readable_to_append['IPV4 ADDRESS'] = asset_details.get('ipv4')
+            human_readable_to_append['OPERATING SYSTEM'] = asset_details.get('operating_system')
+            human_readable_to_append['SYSTEM TYPE'] = asset_details.get('device_type')
+            human_readable_to_append['DNS NAME (FQDN)'] = asset_details.get('fqdn')
+        if plugin_details:
+            human_readable_to_append['PLUGIN ID'] = plugin_details.get('id')
+            human_readable_to_append['PLUGIN NAME'] = plugin_details.get('name')
+            human_readable_to_append["VULNERABILITY PRIORITY RATING"] = (
+                plugin_details.get("vpr").get("score") if plugin_details.get("vpr") else None
+            )
+            human_readable_to_append['CVSSV2 BASE SCORE'] = plugin_details.get('cvss_base_score')
+            human_readable_to_append['CVE'] = plugin_details.get('cve')
+            human_readable_to_append['DESCRIPTION'] = plugin_details.get('description')
+            human_readable_to_append['SOLUTION'] = plugin_details.get('solution')
+        if port_details:
+            human_readable_to_append['PORT'] = port_details.get('port')
+            human_readable_to_append['PROTOCOL'] = port_details.get('protocol')
+
+        human_readable_to_append['SEVERITY'] = chunk_details.get('severity')
+        human_readable_to_append['FIRST SEEN'] = chunk_details.get('first_found')
+        human_readable_to_append['LAST SEEN'] = chunk_details.get('last_found')
+
+        remove_nulls_from_dictionary(chunk_details)
+        remove_nulls_from_dictionary(human_readable_to_append)
+        human_readable.append(human_readable_to_append)
+    return CommandResults(
+        outputs_prefix='TenableIO.Vulnerability',
+        outputs=chunks_details_list,
+        raw_response=chunks_details_list,
+        readable_output=tableToMarkdown('Export Vulnerabilities Results', human_readable, headers=headers, removeNull=True)
+    )
+
+
+def validate_range(range: Optional[str]) -> tuple[Optional[float], Optional[float]]:
+    """
+    Validates the vprScoreRange argument for export asset command
+    Args:
+        range (str): A str represents a range for example 3-5.
+    Returns:
+        Range if valid else raise DemistoException.
+    """
+    if range:
+        range_without_spaces = range.strip()
+        nums_list = range_without_spaces.split("-")
+        if len(nums_list) < 2:
+            raise DemistoException('Please specify valid vprScoreRange')
```

Comment:
What's the valid score range?

Reply:
changed to Please specify valid vprScoreRange. For example: 3.5-5.5.,